ts/store/actions/persistedPreferences.ts

```diff
@@ -16,8 +16,14 @@ export const serviceAlertDisplayedOnceSuccess = createStandardAction(
   "SERVICE_ALERT_DISPLAYED_ONCE_SUCCESS"
 )<{ wasServiceAlertDisplayedOnce: boolean }>();
 
+export const preferencesExperimentalFeaturesSetEnabled = createStandardAction(
+  "PREFERENCES_EXPERIMENTAL_FEATURES_SET_ENABLED"
```

Comment:
@chiadav you can add the pagopa action here too instead of creating a new action file